docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+   | RRFS_CONUS_25km      | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+   | RRFS_CONUS_13km      | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+   | RRFS_CONUS_3km       | ESG grid          | lambert_conformal              |
+   +----------------------+-------------------+--------------------------------+
+
+
+.. _GenerateForecast:
+
+Generate the Forecast Experiment 
+=================================
+Generating the forecast experiment requires three steps:
+
+* :ref:`Set experiment parameters <ExptConfig>`
+* :ref:`Set Python and other environment parameters <SetUpPythonEnv>`
+* :ref:`Run a script to generate the experiment workflow <GenerateWorkflow>`
+
+The first two steps depend on the platform being used and are described here for each Level 1 platform. Users will need to adjust the instructions to their machine if they are working on a Level 2-4 platform. Information in :numref:`Chapter %s: Configuring the Workflow <ConfigWorkflow>` can help with this. 
+
+.. _ExptConfig:
+
+Set Experiment Parameters
+---------------------------- 
+
+Each experiment requires certain basic information to run (e.g., date, grid, physics suite). This information is specified in ``config_defaults.sh`` and in the user-specific ``config.sh`` file. When generating a new experiment, the SRW App first reads and assigns default values from the ``config_defaults.sh`` file. Then, it reads and (re)assigns variables from the user's custom ``config.sh`` file. For background info on ``config_defaults.sh``, read :numref:`Section %s <DefaultConfigSection>`, or jump to :numref:`Section %s <UserSpecificConfig>` to continue configuring the experiment. 
+
+.. _DefaultConfigSection:
+
+Default configuration: ``config_defaults.sh``
+------------------------------------------------
+
+.. note::
+   This section provides background information on how the SRW App uses the ``config_defaults.sh`` file. This information is helpful but not essential to running the SRW App. Users may skip to :numref:`Step %s <UserSpecificConfig>` to continue configuring their experiment. 
+
+Important configuration variables in the ``config_defaults.sh`` file appear in 
+:numref:`Table %s <ConfigVarsDefault>`. Some of these default values are intentionally invalid in order to ensure that the user assigns valid values in the user-specified ``config.sh`` file. Any settings provided in ``config.sh`` will override the default ``config_defaults.sh`` 
+settings. There is usually no need for a user to modify the default configuration file. Additional information on the default settings can be found in the file itself and in :numref:`Chapter %s <ConfigWorkflow>`. 
+
+.. _ConfigVarsDefault:
+
+.. table::  Configuration variables specified in the config_defaults.sh script.
```

Comment:
There will be new ones that will need to be added to the list for the verification tasks. They are in the Google Doc I pointed you to but we can merge those in another PR. Just wanted to mention it here as a reminder.

Reply:
Would they go here or in the chapter about config_defaults.sh and config.sh? Regardless, I've made note to address that later.

Reply:
Likely in the config_defaults chapter.